pkg/synchronizer/track.go

```diff
@@ -139,6 +139,10 @@ func (t *TrackSynchronizer) OnSenderReport(f func(drift time.Duration)) {
 func (t *TrackSynchronizer) PrimeForStart(pkt jitter.ExtPacket) ([]jitter.ExtPacket, int, bool) {
 	if t.initialized || t.startGate == nil {
 		if !t.initialized {
+			// skip dummy packets
+			if len(pkt.Payload) == 0 {
+				return nil, 0, false
+			}
```

Comment:
I am thinking this can be dropped even if `startGate != nil`.

I think `startGate` will drop it based on same time stamp, but I think it will be better to drop here explicitly till initialised irrespective of whether start gate is there or now.